src/reducers/versions.tsx

```diff
@@ -1523,17 +1542,35 @@ export const createReducer = ({
       }
       case getType(actions.setCurrentVersionId): {
         const { versionId } = action.payload;
-        let { expandedPaths } = state;
+        let { expandedPaths, versionInfo } = state;
+        const { selectedPath, visibleSelectedPath } = state;
         const version = getVersionInfo(state, versionId);
 
         if (version && expandedPaths) {
           expandedPaths = _getPrunedExpandedPaths(expandedPaths, version);
         }
 
+        // TODO: remove this conditional statement once no code depends
+        // on `selectedPath` and `visibleSelectedPath` in version
+        // See https://github.com/mozilla/addons-code-manager/issues/1258
+        /* istanbul ignore next */
+        if (version) {
```

Comment:
This is only for syncing `version.selectedPath` and `version.visibleSelectedPath` with the new props out of version object, so that I will not break all components depending on these two props.